app/src/main/java/fr/free/nrw/commons/upload/mediaDetails/UploadMediaDetailFragment.java

```diff
@@ -261,6 +262,12 @@ public void onImageValidationSuccess() {
         callback.onNextButtonClicked(callback.getIndexInViewFlipper(this));
     }
 
+    @Override
```

Comment:
We have a rule that you must add javadoc for all new methods (even overrides). It can be very simple, just say when this method is called. Thanks!